spec/tests/inverted-index-test.js

```diff
@@ -1,5 +1,144 @@
-describe('check book length', () => {
-    it('book should not be empty', () => {
-        expect(books.length > 0).toBeTruthy();
+const books = require('../books.json');
+const adventureBooks = require('../adventure-books.json');
+const emptyArray = require('../empty.json');
+const invalidContent = require('../invalid-content.json');
+const invalidFile = require('../invalid-file.json');
+const invalidKey = require('../invalid-key.json');
+
+const invertedIndex = new InvertedIndex();
+
+describe('Files to be indexed', () => {
+  it('should have keys named `title` and `text` with string for values', () => {
+    expect(invertedIndex.validateFile(invalidContent)).toBe('Invalid file content');
+    expect(invertedIndex.validateFile(invalidKey)).toBe('Invalid file content');
+  });
+  it('should not be an empty file', () => {
+    expect(invertedIndex.validateFile(emptyArray)).toBe('Empty file');
+  });
+  it('should not be an invalid file', () => {
+    expect(invertedIndex.validateFile(books)).toBe('Valid file');
+  });
+  it('should be an invalid file', () => {
+    expect(invertedIndex.validateFile(invalidFile)).toBe('Invalid file');
+  });
+});
+
+describe('JSON file tokenized', () => {
+  const bookToTokenize = [{ title: 'Alice , / ?', text: 'enters a a.' }];
+
+  it('should return " array " for a valid json file input', () => {
+    expect(typeof (InvertedIndex.tokenize(`${bookToTokenize[0].title} ${bookToTokenize[0].text}`))).toBe(typeof ([]));
+  });
+  it('should return "an array of books with filtered contents"', () => {
+    expect(InvertedIndex.tokenize(`${bookToTokenize[0].title} ${bookToTokenize[0].text}`)).toEqual(
+      ['alice', 'enters', 'a']
+    );
+  });
+});
+
+describe('Get index of a JSON file', () => {
+  invertedIndex.createIndex(books, 'books.json');
+  invertedIndex.createIndex(adventureBooks, 'adventure-books.json');
+  it('should return `undefined` if index does not exist', () => {
```

Comment:
I believe you should have separate test suites to test `createIndex` and `getIndex` and not have them together in the same suite.